compiler/luci/lang/include/luci/IR/Module.h

```diff
@@ -59,8 +60,27 @@ class Module final
 
   // TODO provide graph accessor with a name
 
+public:
+  void source_table(const std::map<uint32_t, std::string> &table) { _source_table = table; }
+
+  const std::map<uint32_t, std::string> &source_table(void) const { return _source_table; }
+
 private:
   std::vector<std::unique_ptr<loco::Graph>> _graphs;
+
+private:
+  /**
+   * @brief Metadata about source table for profiling
+   *
+   * @note  Key is ID of node and value is name of node.
+   *
+   *        If there was originally imported 'source_table' in circle model,
+   *        the table will be stored as it is.
+   *        Otherwise, new 'source_table' is created with imported nodes.
+   *
+   *        Only supported for a module with single subgraph.
```

Comment:
Q) In draft and in this PR, there is no validation check for this condition. Will it be in other codes?

Reply:
Oh... the comment is not proper... I will fix it, thank you :)

My intention was "Only first subgraph is considered even if `Module` has multiple subgraphs"